FirebaseAdmin/FirebaseAdmin/Messaging/TopicManagementResponse.cs

```diff
@@ -25,21 +26,26 @@ public sealed class TopicManagementResponse
         /// <summary>
         /// Initializes a new instance of the <see cref="TopicManagementResponse"/> class.
         /// </summary>
-        /// <param name="instanceIdServiceResponse">The results from the response produced by FCM topic management operations.</param>
-        public TopicManagementResponse(InstanceIdServiceResponse instanceIdServiceResponse)
+        /// <param name="topicManagementResults">The results from the response produced by FCM topic management operations.</param>
+        public TopicManagementResponse(List<string> topicManagementResults)
```

Comment:
Ok, I see why `InstanceIdServiceResponse` needs to be visible. Lets make that `internal` and change this constructor back to accept an `InstanceIdServiceResponse`.